app/services/invoices/generate_pdf_service.rb

```diff
@@ -42,9 +42,13 @@ def generate_pdf
 
     def template
       if invoice.one_off?
-        'invoices/one_off'
+        return "invoices/v3/one_off" if invoice.version_number < 4
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)